packages/@okta/vuepress-site/docs/release-notes/2020-09-1/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.09.1
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2020.09.1](#bugs-fixed-in-2020-09-1)       | September 10, 2020           |
+
+### Bugs fixed in 2020.09.1
+
+* When attempting to reset a user's password using the [`lifecycle/reset_password` endpoint](/docs/reference/api/users/#reset-password), admins received a 500 `Internal Server` error code rather than a valid error message if the user's email address was invalid. (OKTA-307089)
```

Comment:
```suggestion
* When attempting to reset a user's password using the [`lifecycle/reset_password` endpoint](/docs/reference/api/users/#reset-password), admins received an HTTP 500 error code rather than a valid error message if the user's email address was invalid. (OKTA-307089)
```

Reply:
Fixed.